Gemfile.lock

```diff
@@ -21,6 +22,8 @@ GEM
       archive-tar-minitar
       excon (>= 0.28)
       json
+    etcd (0.2.4)
```

Comment:
So while testing this I came across quite a pickle, which is that the etcd ruby gem < 0.3.0 cannot handle etcd 2.0+ (it errors all over the place regarding 404 errors)

https://github.com/ranjib/etcd-ruby/pull/51 has been merged and my local testing indicates that version 0.3.0 seems to fix things.

@bobtfish do you think we should depend on an old etcd or the new one? I sorta prefer the new one but I'm not sure what the community progress on moving to etcd 2.0 is like?